ansible/roles/mailman3/templates/etc/ansible/facts.d/mailman3.fact.j2

```diff
@@ -0,0 +1,20 @@
+#!{{ ansible_python['executable'] }}
+
+# Copyright (C) 2020 CipherMail B.V. <https://www.ciphermail.com/>
+# Copyright (C) 2020 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-only
+
+# {{ ansible_managed }}
+
+from json import dumps
+import apt
+
+cache = apt.Cache()
+output = {}
+
+try:
+    output['installed'] = cache['mailman3'].is_installed
```

Comment:
Do you know which method of checking if package is installed (either via APT cache or via presence in `$PATH`) is faster and/or less taxing on the system in question? Keep in mind that this script will be executed each time Ansible gets the facts which happens frequently in DebOps.

Reply:
Let's go with the `$PATH` approach:
```
imre@lists2:/etc/ansible/facts.d$ time ./mailman3-apt.fact
{
    "installed": true
}

real	0m0,364s
user	0m0,327s
sys	0m0,035s
imre@lists2:/etc/ansible/facts.d$ time ./mailman3-path.fact
{
    "installed": true
}

real	0m0,047s
user	0m0,030s
sys	0m0,017s
```